contributing.md

```diff
@@ -220,6 +220,53 @@ You may want to read this for a more detailed explanation (and links to other po
 how to write a good commit message). This content is licensed under 
 [CC-BY-SA](https://creativecommons.org/licenses/by-sa/4.0/).
 
+### Sign off your commit. 
+
+You need to sign off your commit. By adding your 'Signed-off-by' line to the commit messages you adhere to Developer Certificate of Origin requirements. Here is the text of the [DCO](https://developercertificate.org/): 
```

Comment:
Thank you. Please wrap the line at 88 characters. It might also be possible to shorten it a little and adding the git command to the top might make it more likely to be noticed

```suggestion
You need to sign off your commit. By adding your 'Signed-off-by' line to the commit messages you adhere to the [Developer Certificate of Origin (DCO)](https://developercertificate.org/).

Use the `-s` command-line option to append the `Signed-of-by` line when committing your code:
￼
￼`$ git commit -s`

This is the full text of the DCO:
```

What do you think?